src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,40 @@
+package calculator
+
+class Calculator {
+    fun calculate(input: String): Int {
+        val numbers = extractNumbers(input)
+        return doAddition(numbers)
+    }
+
+    private fun extractNumbers(input: String): List<String> {
+        return if (input.startsWith(CUSTOM_DELIMITER_START_SIGN)) {
+            extractCustomCase(input)
+        } else {
+            extractDefaultCase(input)
+        }
+    }
```

Comment:
```suggestion
    private fun extractNumbers(input: String): List<String> {
        return when (input.startsWith(CUSTOM_DELIMITER_START_SIGN)) {
                true -> extractCustomCase(input)
                false -> extractDefaultCase(input)
            }
    }
    
    private fun extractNumbers(input: String): List<String> {
        return when {
                input.startsWith(CUSTOM_DELIMITER_START_SIGN) -> extractCustomCase(input)
                else -> extractDefaultCase(input)
            }
    }
```
요런 코드도 가능할거 같아요~ `when`

Reply:
오!!! 이거 좋네요 👍🏻👍🏻